src/main/java/com/minecolonies/coremod/generation/defaults/DefaultResearchProvider.java

```diff
@@ -76,8 +76,9 @@ public Collection<ResearchEffect> getResearchEffectCollection()
         effects.add(new ResearchEffect(BLOCK_ATTACKS).setTranslatedName("Knight Shield Blocking Chance +%3$s%%").setLevels(new double[] {0.05, 0.1, 0.25, 0.5}));
         effects.add(new ResearchEffect(BLOCK_BREAK_SPEED).setTranslatedName("Citizen Block Break Speed +%3$s%%").setLevels(new double[] {0.1, 0.25, 0.5, 1, 2}));
         effects.add(new ResearchEffect(BLOCK_PLACE_SPEED).setTranslatedName("Citizen Block Place Speed +%3$s%%").setLevels(new double[] {0.1, 0.25, 0.5, 1, 2}));
-        effects.add(new ResearchEffect(CITIZEN_CAP).setTranslatedName("Increase Max Citizens to %s")
-                      .setLevels(new double[] {CitizenConstants.CITIZEN_LIMIT_OUTPOST, CitizenConstants.CITIZEN_LIMIT_HAMLET, CitizenConstants.CITIZEN_LIMIT_VILLAGE,
+        effects.add(new ResearchEffect(CITIZEN_CAP).setTranslatedName("Increase Max Citizens by %s")
```

Comment:
It might be a little more consistent with the other effect names to use `Max Citizens +%s` here instead.